Resources/Prototypes/_RMC14/Entities/Mobs/NPCs/animals.yml

```
@@ -18,6 +18,17 @@
   components:
   - type: GhostRole
     prob: 0
+  - type: Respirator
+    damage:
+      types:
+        Asphyxiation: 0
+    damageRecovery:
+      types:
+        Asphyxiation: 0
+  - type: Barotrauma
+    damage:
+      types:
+        Blunt: 0
```

You should be able to replace all of these with {} for an empty damage specifier
```suggestion
  - type: Respirator
    damage: {}
    damageRecovery: {}
  - type: Barotrauma
    damage: {}
```